modules/passkey/test/utils/safe.ts

```diff
@@ -0,0 +1,43 @@
+import { ethers } from 'hardhat'
```

Comment:
I think this isn't used anymore correct?

Reply:
It is used for `ethers.ZeroAddress`.

Reply:
I mean the `modules/passkey/test/utils/safe.ts` file.

Reply:
Removed 👍🏾